src/main/kotlin/com/sourcegraph/cody/agent/protocol/InlineAutocompleteList.kt

```diff
@@ -0,0 +1,8 @@
+package com.sourcegraph.cody.agent.protocol
+
+import com.sourcegraph.cody.vscode.InlineAutocompleteItem
+
+class InlineAutocompleteList {
+  val items: List<InlineAutocompleteItem>? = null
```

Comment:
This can be non-null, according to agent-protocol.ts

Reply:
so `emptyList()` default is ok, right?

Reply:
Yup thats correct